### PR TITLE
Feature/improve issue parsing performance

### PR DESCRIPTION
**Pull request by catcombo**

### CODE REVIEW

File: youtrack_sdk/client.py [hunk 1]
```diff
@@ -109,13 +109,13 @@ def _send_request(
             return
 
         try:
-            return response.json()
+            return response.content
         except JSONDecodeError as e:
             raise YouTrackException(
                 f"Failed to decode response from {method} {url}, status={response.status_code}",
             ) from e
```
Comment (mm-matthias) [inline youtrack_sdk/client.py:116]: I think the try/except-block does not make sense anymore.

**Review by mm-matthias — COMMENTED** [review]
LGTM in general, added one comment.

File: youtrack_sdk/client.py (hunk 1 above)
Comment (catcombo) [reply youtrack_sdk/client.py]: You're right! Thank you. Updated.

**Pull request merged**